src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const utils = require('./utils');
+const removeUndefined = require('./parse/utils').removeUndefined;
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return removeUndefined({
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
```

Comment:
I see what you're saying but I would probably just have went with 1 getAccount*, with {settings: {...}, properties: {...}} somewhat inline with what you had now.